### ISSUE TITLE
When running focused specs output shows other specs as passed

### ISSUE THREAD

**Issue by atermenji** [angular/protractor#1683]
When using `ddescribe()` or `iit()` protractor runs only focused specs (I can see it in browser), but in the output it shows that all other specs passed. 

For example, if I have 1 `describe()` with 9 `it()`s and 1 `iit()`   I would expect to see something like:
```
1 example, 1 passed, 0 failures
```
but instead I see:
```
10 examples, 10 passed, 0 failures
```

**Comment by hankduan** [comment]
This is an issue with Jasmine 1.3. Protractor simply reports the results from Jasmine and there is nothing Protractor can/should do to fix this.
That being said, it is fixed in Jasmine 2.0; follow this for protractor's status for supporting jasmine 2.0: https://github.com/angular/protractor/issues/362

**Comment by atermenji** [comment]
@hankduan Thanks!
